README.md

```diff
@@ -1,2 +1,101 @@
-# community
-TiDB community content
+# TiDB Community Readme
+
+Welcome to the TiDB Community! The main objective is to help members of the TiDB community who share similar interests to learn from and collaborate with each other.
+
+Your journey of becoming a contributor and committer starts from here: improving docs, improving code, giving talks, organizing meetups, etc.
+
+Love you all.
+
+# Communicating
+
+You can reach out to the community members via any one of the following ways: 
+
+- Google Groups：[https://groups.google.com/forum/#!forum/tidb-user](https://groups.google.com/forum/#!forum/tidb-user)
+- Stackoverflow：[https://stackoverflow.com/questions/tagged/tidb](https://stackoverflow.com/questions/tagged/tidb)
+- Twitter: [https://twitter.com/PingCAP](https://twitter.com/PingCAP)
+- Facebook: [https://www.facebook.com/pingcap2015](https://www.facebook.com/pingcap2015)
+- Reddit：[https://www.reddit.com/r/TiDB](https://www.reddit.com/r/TiDB)
+
+# How Can I Help?
+
+- Try [TiDB](https://github.com/pingcap/tidb)! If you like it, star, share, and recommend it. 
+- To dig deeper, read a design doc, e.g. [architecture](https://www.pingcap.com/blog/2017-05-23-perconalive17), [How we build TiDB](https://www.pingcap.com/blog/2016-10-17-how-we-build-tidb)
+- Documentation can always use improvement!
```

Comment:
Documentation is always welcomed!

Reply:
Vote for your sentence, I will change it for easy to read.